ui/nuxeo-data-table/data-table-column-sort.js

```diff
@@ -52,13 +52,7 @@ import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
         </style>
 
         <div style="position: relative">
-          <paper-icon-button
-            id="sortIcon"
-            on-click="_sort"
-            icon="data-table:arrow-upward"
-            direction$="[[direction]]"
-            aria-label$="[[i18n('command.sort')]]"
-          >
+          <paper-icon-button id="sortIcon" on-click="_sort" icon="data-table:arrow-upward" direction$="[[direction]]">
```

Comment:
Instead of calling `_computeAriaLabel` from both `ready` and `_order` would it be easier to use a single computed binding instead?
```suggestion
          <paper-icon-button id="sortIcon" on-click="_sort" icon="data-table:arrow-upward" direction$="[[direction]]" aria-label$="[[_computeAriaLabel(direction, i18n)]]">
```